apps/docs/src/app/core/component-docs/info-label/info-label-docs.component.html

```diff
@@ -8,7 +8,7 @@
 <label fd-form-label>Default Info label by passing the default message </label>
 <separator></separator>
 
-<fd-docs-section-title [id]="'filled'" [componentName]="'infolabel'">
+<fd-docs-section-title [id]="'infoLabelText'" [componentName]="'infolabel'">
 	Info label with text
 </fd-docs-section-title>
 <description> <code>fd-info-label fd-info-label--accent-color-1</code> modifier can be used to render a Text version of the Info label. </description>
```

Comment:
The `fd-info-label`and `fd-info-label--accent-color-1` modifiers can be used to render a text version of the Info Label.